### PR TITLE
Added initial outputs and updated the public/private subnets 4more cidrs

### PR DESCRIPTION
**Pull request by Vermyndax**
Added initial outputs and updated the public/private subnets to support multiple cidrs for multiple subnets, just in case. Portability.

### CODE REVIEW

File: terraform/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  value = "${var.aws_az2}"
+}
+
+output "mgmt_vpc_name" {
+  value = "${var.mgmt_vpc_name}"
+}
+
+output "mgmt_vpc_id" {
+  value = "${module.mgmt-vpc.vpc_id}"
+}
```
Comment (afeld) [inline terraform/outputs.tf:19]: Do dashes work in Terraform references? Seems to break the syntax highlighter, at least. I'd suggest using underscores, for consistency.

File: terraform/variables.tf [hunk 2]
```diff
@@ -13,11 +13,11 @@ variable "mgmt_vpc_name" {
...
-variable "mgmt_public_subnet_cidr" {
-  type = "string"
+variable "mgmt_public_subnet_cidrs" {
+  type = "list"
 }
-variable "mgmt_private_subnet_cidr" {
-  type = "string"
+variable "mgmt_private_subnet_cidrs" {
+  type = "list"
 }
```
Comment (afeld) [inline terraform/variables.tf:21]: Can we add some reasonable defaults here?

**Review by afeld — APPROVED** [review]
Merging now since I want to use these changes sooner than later, but let's address the feedback in follow-up pull request(s).

**Comment by Vermyndax** [comment]
Yep, we can add some defaults... let's create an issue for that and I'll take care of it as I work through the stack.

**Comment by afeld** [comment]
I'll take care of it as part of https://trello.com/c/s22CKJcD/77-integrate-devsecops-infrastructure-and-devsecops-example.

**Pull request merged**